wordpress-mobile/WordPress-iOS
# Jetpack App (Emphasis): Improve Server Credentials Messaging

Fixes https://github.com/wordpress-mobile/WordPress-iOS/issues/16513

## Description

This PR updates the server credentials warning button design on the Restore Options, Scan, and Threat Detail screens.
- Added the plus-small gridicon
- Highlighted the string `Enter your server credentials` so it stands out more

_Top row: server credentials not configured, Bottom row: server credentials configured_

![Simulator Screen Shot - iPhone 12 Pro - 2021-05-27 at 16 36 48](https://user-images.githubusercontent.com/6711616/119791051-6c75b480-bf0f-11eb-9b1d-7155fc3f3128.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-05-27 at 16 36 59](https://user-images.githubusercontent.com/6711616/119791059-6ed80e80-bf0f-11eb-9fec-e3ac142c0e45.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-05-27 at 16 37 09](https://user-images.githubusercontent.com/6711616/119791067-70a1d200-bf0f-11eb-8a55-e034953d5242.png)
-- | -- | --
![Simulator Screen Shot - iPhone 12 Pro - 2021-05-27 at 16 37 39](https://user-images.githubusercontent.com/6711616/119791071-713a6880-bf0f-11eb-975b-1daaba727346.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-05-27 at 16 38 14](https://user-images.githubusercontent.com/6711616/119791075-726b9580-bf0f-11eb-98ed-7f8af7ac2b8a.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-05-27 at 16 38 17](https://user-images.githubusercontent.com/6711616/119791076-73042c00-bf0f-11eb-961d-cfac45521aa2.png)

<details>
 <summary>Expand for WordPress screenshots</summary>
 
 ![Simulator Screen Shot - iPhone 12 Pro - 2021-05-27 at 17 32 50](https://user-images.githubusercontent.com/6711616/119793617-bb244e00-bf11-11eb-9680-fa1c35c61685.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-05-27 at 17 33 20](https://user-images.githubusercontent.com/6711616/119793624-bc557b00-bf11-11eb-92f0-28c609401b62.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-05-27 at 17 33 23](https://user-images.githubusercontent.com/6711616/119793631-bcee1180-bf11-11eb-980d-ca069dd797f5.png)
-- | -- | --

</details>

## How to test
1. Switch to a site without server credentials configured / backup and scan enabled / threats present
2. Go to My Site > Backup > ellipsis icon > Restore
3. ✅ The warning button should be displayed. Also, there should be a plus icon and only the `Enter your server credentials` should be highlighted
4. Go to My Site > Scan
5. ✅ The warning button should be displayed with the updated design
6. Tap on a fixable threat and go to Threat Details
7. ✅ The warning button should be displayed with the updated design

## Regression Notes
1. Potential unintended areas of impact
WordPress: Restore Options, Scan, and Threat Detail screens

2. What I did to test those areas of impact (or what existing automated tests I relied on)
I built and tested the WP app

3. What automated tests I added (or what prevented me from doing so)
N/A, only visual changes

PR submission checklist:
